domains/withoutminh.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "withoutminh"
+  },
+  "records": {
+    "CNAME": "https://guns.lol/withoutminh"
+  },
+  "proxied": true
```

you cant proxied if you are going to use URL records
```suggestion
  }
```